app/helpers/form_helper.rb

```diff
@@ -239,12 +239,12 @@ def counter_f(f, attr, options = {})
     react_form_input('counter', f, attr, options)
   end
 
-  def submit_or_cancel(f, overwrite = false, args = { })
+  def submit_or_cancel(f, overwrite = false, args = { }, button_text = "Submit")
     args[:cancel_path] ||= resource_path(controller_name)
     cancel_btn = args[:react_cancel_button] ? react_cancel_button(args) : link_to(_("Cancel"), args[:cancel_path], :class => "btn btn-default")
     content_tag(:div, :class => "clearfix") do
       content_tag(:div, :class => "form-actions") do
-        text    = overwrite ? _("Overwrite") : _("Submit")
+        text    = overwrite ? _("Overwrite") : _(button_text)
```

Comment:
That's a not quite right way to translate strings, `_()` method should be used on the strings, not on the variables.